os/arch/arm/src/imxrt/imxrt_norflash.c

```diff
@@ -725,6 +702,7 @@ static inline void flexspi_clock_init(void)
 /************************************************************************************
  * Public Functions
  ************************************************************************************/
+#if !defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
```

Comment:
Why did you add this..? This function seems to be used only for imxrt1020 board...